train.py

```diff
@@ -25,12 +25,12 @@
 random.seed(seed)
 np.random.seed(seed)
 
-
+# Training of model
```

Comment:
Providing comments at the top of the function is not a standard way, it's written at the start of the file. Refer to my other scripts. Refer to this tutorial https://realpython.com/documenting-python-code/